scripts/test-with-coverage.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
```

Comment:
```suggestion
#!/usr/bin/env sh
```

nit - can we stick to posix sh please, unless we really need a bashism here.

Reply:
Most scripts around Istio are bash, so probably for consistency sake it should be bash. Doesn't matter too much either way.

Reply:
Yep that's fair. I've encountered bugs using scripts that assume bash in environments that don't have bash. And if the script doesn't actually need bash, it's generally pointless and also less compatible to require it.